docs/desi/desi.rst

```diff
@@ -0,0 +1,111 @@
+.. _astroquery.desi:
+
+************************************
+DESI LegacySurvey Queries (`astroquery.desi`)
+************************************
+
+Getting started
+===============
+
+This module provides a way to query the DesiLegacySurvey service.
+Presented below are examples that illustrate the different types of queries
+that can be formulated.
+
+Query a region
+===============
+
+This example shows how to query a certain region with DesiLegacySurvey.
+We'll use a set of coordinates that define the region of interest,
+and search within a 5 arcmin radius.
+
+.. code-block:: python
```

Comment:
Use the `.. doctest-remote-data::` directive instead of `code-block`, here and everywhere below, otherwise all these make the tests fail.